src/screens/Home/index.tsx

```diff
@@ -30,11 +36,18 @@ const Home = () => {
 
   return (
     <View style={styles.container}>
-      <FlatList
-        data={people}
-        renderItem={renderItem}
-        keyExtractor={keyExtractor}
-      />
+      {isError && <Button title="Try Again" onPress={Home} />}
```

Comment:
si quieres hacer que cuando el usuario presione este boton el request se envie otravez, tienes que "exportar" `fetchData` de `usePeople`:

```
// usePeople.ts

const usePeople = (): UsePeople {
 ...
 return {
    people,
    isLoading,
    error,
    fetchData // <---- esto
 }
}

// Home/index.tsx

const Home = () => {
  const { people, error, loading, fetchData: refetch } = usePeople();
}

```

y despues cambias esto:
```suggestion
      {isError && <Button title="Try Again" onPress={refetch} />}
```

`{ fetchData: refetch } = usePeople()` es como le cambias el nombre a las variables cuando haces "destrocturing" por ejemplo: `{ isLoading: isPeopleLoading } = usePeople()`

Reply:
Hay varios errores cuando intento hacer export de fetchData, tengo que agregarle un type y aja, consegui "Function" que no se si es correcto, pero igual el lint me saca un error en el return line.

 return {
    people,
    isLoading,
    error,
    fetchData(), // <---- aqui
 }

Luego en // Home/index.tsx, cuando cambio el OnPress para que haga refetch me dice que no puede hacer handle de ese type tampoco.